exercises/anagram/test/Tests.hs

```diff
@@ -14,9 +15,14 @@ specs = describe "anagram" $
           describe "anagramsFor" $ for_ cases test
   where
 
-    test Case{..} = it description $ expression `shouldBe` expected
+    test Case{..} = it description $ expression `shouldMatchList` expected
       where
-        expression = anagramsFor subject candidates
+        expression = map toList
+                   . toList
+                   . anagramsFor (fromList subject)
+                   . fromList
+                   . map fromList
```

Comment:
note for my study - this allows either `Text` or `String` to be used. Interesting.